### ISSUE TITLE
Bug: UnidentifiedImageError

### ISSUE THREAD

**Issue by Area0077** [nathom/streamrip#160]
## Bug Report

**Current Behavior**
If an album or single does not have a picture at the source, the downloading halts completely with an error. Downloading was successful till a point where there was an image error

**Command used:**

```bash
# Type the command that caused the bug here
rip url https://open.qobuz.com/artist/161143

**Expected behavior**
downloading should continue regardless of image error from download source?

Error message is listed below, where it's failed and stopped downloading.

Downloading Those About To Rock (Vince Anthony Remix) (Album)

Downloading cover art (large)
Cover |                                                                                                                                                                     | ? left at ?B/s

  UnidentifiedImageError

  cannot identify image file '\\AppData\\Local\\Temp\\cover_-8498833041942376852.jpg'

  at \local\programs\python\python39\lib\site-packages\PIL\Image.py:3023 in open
      3019│     if exclusive_fp:
      3020│         fp.close()
      3021│     for message in accept_warnings:
      3022│         warnings.warn(message)
    → 3023│     raise UnidentifiedImageError(
      3024│         "cannot identify image file %r" % (filename if filename else fp)
      3025│     )
      3026│
      3027│

If this was unexpected, please open a Bug Report at https://github.com/nathom/streamrip/issues/new/choose

**Comment by nathom** [comment]
Fixed in 3eeb9d3b7ef6518936ce6346ce31c8418a8e5022.

**Comment by nathom** [comment]
Fixed in `v1.5`.
